{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -62,3 +63,4 @@ repos:
       language: system
       files: '(README\.rst)'
       name: restructuredtext-lint - Is the README PyPi-compliant?
```

Comment:
Disclaimer: I have not checked. Just out of curiosity: are the rules of PyPi particularly useful for the use case of a research project?

Reply:
Actually I don't know. I would not need it. Was already in there so I did not delete it. But I would not use it.

Reply:
Okay, let's get rid of it then. Less maintenance.